keyboards/bastardkb/dilemma/3x5_3/rules.mk

```diff
@@ -25,5 +29,7 @@ POINTING_DEVICE_DRIVER = cirque_pinnacle_spi
 
 SPLIT_KEYBOARD = yes
 
+DEBOUNCE_TYPE = asym_eager_defer_pk
+
```

Comment:
I'm wondering if we should leave this sort of optimization to the user's discretion.

The rational being that the stock keymaps should showcase the keyboard's feature, but use as much of the defaults as possible.

What do you think?